### PR TITLE
Consistent error logging in standalone registry/IP allocator

### PR DESCRIPTION
**Pull request by pravisankar**

### CODE REVIEW

**Comment by pravisankar** [comment]
@rajatchopra @danwinship PTAL

File: ovssubnet/registry/registry.go [hunk 1]
```diff
@@ -59,7 +58,7 @@ func newNodeEvent(action, key, value string) *api.NodeEvent {
 		}
 	}
 
-	fmt.Printf("Error decoding node event: nil key (%s,%s,%s).\n", action, key, value)
+	log.Errorf("Error decoding node event: nil key (%s,%s,%s).\n", action, key, value)
```
Comment (danwinship) [inline ovssubnet/registry/registry.go]: The glog functions don't require a trailing "\n". We might as well be consistent about that too.

File: pkg/netutils/ip_allocator.go [hunk 2]
```diff
@@ -20,11 +22,11 @@ func NewIPAllocator(network string, inUse []string) (*IPAllocator, error) {
 	for _, netStr := range inUse {
 		_, nIp, err := net.ParseCIDR(netStr)
 		if err != nil {
-			fmt.Println("Failed to parse network address: ", netStr)
+			log.Errorf("Failed to parse network address: ", netStr)
```
Comment (danwinship) [inline pkg/netutils/ip_allocator.go]: need to either use Errorln or else a %s

**Comment by pravisankar** [comment]
@danwinship Updated

**Pull request merged**